application.example.yaml

```diff
@@ -166,9 +166,12 @@ trading:
         - REP/USD
         - ZEC/USD
 
-      # The amount of fees Kraken charges. If an exchange's API supports requesting fees at runtime, the bot
+      # The amount of fees Kraken charges for long trades. If an exchange's API supports requesting fees at runtime, the bot
```

Comment:
This comment change is not accurate. The 0.0026 `fee` value is applied to all trades. The 0.0002 `marginFee` is applied _in addition to_ `fee` to any margin trades.